src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/ConfirmYourTrainingProvider.cshtml.cs

```diff
@@ -18,37 +21,39 @@ public class ConfirmYourTrainingModel : PageModel
         public string TrainingProviderName { get; set; }
 
         [BindProperty]
-        public bool? ConfirmTrainingProvider { get; set; }
+        public bool? ConfirmedTrainingProvider { get; set; }
 
-        public string Backlink => $"/apprenticeships/{ApprenticeshipId.Hashed}";
+        public string BackLink => $"/apprenticeships/{ApprenticeshipId.Hashed}";
 
         public ConfirmYourTrainingModel(IOuterApiClient client, AuthenticatedUser authenticatedUser)
         {
-            _authenticatedUser = authenticatedUser;
-            _client = client;
+            _authenticatedUser = authenticatedUser ?? throw new ArgumentNullException(nameof(authenticatedUser));
+            _client = client ?? throw new ArgumentNullException(nameof(client));
         }
```

Comment:
Not sure if the constructor will ever be called if the parameters to it are going to be null. but this very much will depend on the IoC we use and also I imagine if a factory is creating a input, but I've just being assuming it won't get called. 

Only because I don't see this very valid check in our general code base

Reply:
I'm pretty sure you're right - the IoC won't allow the object to be created with null values.

The `#nullable` warnings moan if the check isn't present.  I think I would prefer not to have the check to be honest as it's just clutter.  However that will man alternative clutter to suppress the warnings!